torch/_inductor/codegen/cpp.py

```diff
@@ -1720,11 +1784,18 @@ class CppKernel(Kernel):
 
     def __init__(self, args, num_threads):
         super().__init__(args)
+        # Indicate when this kernel is active, for example
+        # {x0, {24, 26}} -> this kernel is activate when x0 >=24 and x0 < 26
+        self.active_ranges: dict[sympy.Expr, Tuple[sympy.Expr, ...]] = {}
+        # Indicate this kernel will be moved under inner for loop
+        # See move_code_under_inner_loop
+        self.inner_itervars: List[sympy.Symbol] = []
         self.call_ranges: Optional[Tuple[sympy.Expr, ...]] = None
         self.ranges: List[sympy.Expr] = []
         self.itervars: List[sympy.Symbol] = []
         self.reduction_depth = None
         self.reduction_prefix = IndentedBuffer()
+        self.reduction_prefix_generators: List[Callable] = []  # type: ignore[type-arg]
```

Comment:
Add code doc to explain why we need this and how the semantics of `Callable`.

Reply:
Added